include/sentry.h

```diff
@@ -834,12 +834,23 @@ SENTRY_API void sentry_options_set_symbolize_stacktraces(
 SENTRY_API int sentry_options_get_symbolize_stacktraces(
     const sentry_options_t *opts);
 
+/**
+ * Adds a new pattern for *in-app* classification.
+ *
+ * When on-device symbolication is enabled (see
+ * `sentry_options_set_symbolize_stacktraces`), this will flag stack frames as
+ * *in-app*, which match this pattern.
+ * Matching is done on both symbols names and object file names they belong to.
+ */
+SENTRY_API void sentry_options_add_in_app_include(
```

Comment:
Please also add `in_app_exclude`. After merging, let's add `native` to supported platforms here:

https://github.com/getsentry/sentry-docs/blob/322a46a6bd98f7f564db74e8d7752312d94c3cee/src/platforms/common/configuration/options.mdx#L157-L171